modular_zubbers/code/modules/quirks/code/negative_quirks/sensative_soles.dm

```diff
@@ -0,0 +1,9 @@
+/datum/quirk/sensative_soles
+	name = "Sensative Soles"
+	desc = "You've never even heard of a callous. Anything but the softest of surfaces will make your feet feel like they're on fire. Shoes do not protect you from this."
+	value = -3
+	mob_trait = TRAIT_SENSATIVE_SOLES
```

Comment:
three separate instances of 'sensative' that should be 'sensitive'
somehow the line directly below this comment got it right

also the name of the file needs to be corrected